github/authorizations.go

```diff
@@ -85,6 +85,20 @@ func (a AuthorizationApp) String() string {
 	return Stringify(a)
 }
 
+// OAuthGrant represents an OAuth application that has been granted access to an account.
+type Grant struct {
```

Comment:
Since the type name is `Grant`, not `OAuthGrant`, the comment should begin with:

```Go
// Grant represents ...
```